<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zelda Map Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Resources/Images/Icons/editer-editeur-papier-crayon-ecrivez-icone-6803-128.png</normaloff>:/Resources/Images/Icons/editer-editeur-papier-crayon-ecrivez-icone-6803-128.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionLawer_layer"/>
    <addaction name="actionHigh_layer"/>
    <addaction name="actionCollision_layer"/>
    <addaction name="separator"/>
    <addaction name="actionVisualization"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionZoom_1_1"/>
    <addaction name="actionZoom_1_2"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Grid"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBrush"/>
    <addaction name="actionPaint_pot"/>
    <addaction name="actionPipette"/>
   </widget>
   <widget class="QMenu" name="menuEvents">
    <property name="title">
     <string>Events</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionChange_dimension"/>
    <addaction name="actionChange_chipset"/>
    <addaction name="actionClear_map"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuEvents"/>
   <addaction name="menuMap"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionLawer_layer"/>
   <addaction name="actionHigh_layer"/>
   <addaction name="actionCollision_layer"/>
   <addaction name="separator"/>
   <addaction name="actionVisualization"/>
   <addaction name="actionShow_Grid"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_1_1"/>
   <addaction name="actionZoom_1_2"/>
   <addaction name="separator"/>
   <addaction name="actionBrush"/>
   <addaction name="actionPaint_pot"/>
   <addaction name="actionPipette"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGraphicsView" name="graphicsViewChipset">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::RubberBandDrag</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/fichier-une-page-nouvelle--icone-4740-48.png</normaloff>:/Resources/Images/Icons/fichier-une-page-nouvelle--icone-4740-48.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/dossier-icone-4159-48.png</normaloff>:/Resources/Images/Icons/dossier-icone-4159-48.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/disque-sauvegardez-icone-4699-48.png</normaloff>:/Resources/Images/Icons/disque-sauvegardez-icone-4699-48.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/que-sauf-icone-5284-48.png</normaloff>:/Resources/Images/Icons/que-sauf-icone-5284-48.png</iconset>
   </property>
   <property name="text">
    <string>Save as ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/porte-sortie-icone-7442-48.png</normaloff>:/Resources/Images/Icons/porte-sortie-icone-7442-48.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/defaire-icone-8681-48.png</normaloff>:/Resources/Images/Icons/defaire-icone-8681-48.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/redo-icone-7712-48.png</normaloff>:/Resources/Images/Icons/redo-icone-7712-48.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionLawer_layer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/couche1.png</normaloff>:/Resources/Images/Icons/couche1.png</iconset>
   </property>
   <property name="text">
    <string>Low layer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionHigh_layer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/couche2.png</normaloff>:/Resources/Images/Icons/couche2.png</iconset>
   </property>
   <property name="text">
    <string>High layer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionCollision_layer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/coucheCollision.png</normaloff>:/Resources/Images/Icons/coucheCollision.png</iconset>
   </property>
   <property name="text">
    <string>Collision layer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionVisualization">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/couches-carte-icone-7284-128.png</normaloff>:/Resources/Images/Icons/couches-carte-icone-7284-128.png</iconset>
   </property>
   <property name="text">
    <string>Visualization</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionZoom_1_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/minimiser-icone-8625-48.png</normaloff>:/Resources/Images/Icons/minimiser-icone-8625-48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom 1:1</string>
   </property>
  </action>
  <action name="actionZoom_1_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/maximiser-icone-5962-48.png</normaloff>:/Resources/Images/Icons/maximiser-icone-5962-48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom 1:2</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/pinceau-icone-4100-128.png</normaloff>:/Resources/Images/Icons/pinceau-icone-4100-128.png</iconset>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
  </action>
  <action name="actionPaint_pot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/paintroller-icone-5978-128.png</normaloff>:/Resources/Images/Icons/paintroller-icone-5978-128.png</iconset>
   </property>
   <property name="text">
    <string>Paint pot</string>
   </property>
  </action>
  <action name="actionPipette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/couleur-couleur-cueilleur-icone-6523-48.png</normaloff>:/Resources/Images/Icons/couleur-couleur-cueilleur-icone-6523-48.png</iconset>
   </property>
   <property name="text">
    <string>Pipette</string>
   </property>
  </action>
  <action name="actionChange_dimension">
   <property name="text">
    <string>Change dimension</string>
   </property>
  </action>
  <action name="actionChange_chipset">
   <property name="text">
    <string>Change chipset</string>
   </property>
  </action>
  <action name="actionClear_map">
   <property name="text">
    <string>Clear map</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShow_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Resources/Images/Icons/grid.png</normaloff>:/Resources/Images/Icons/grid.png</iconset>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear_map</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBrush</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>brushTool()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChange_chipset</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeChipset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCollision_layer</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>collisionLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHigh_layer</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>higherLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLawer_layer</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>lowerLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaint_pot</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>paintPotTool()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPipette</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>pipetteTool()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Grid</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>gridLayer(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_1_1</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>minimizeMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_1_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>maximizeMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVisualization</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>visuaLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeActiveSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose_All</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeAllSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNext</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>activateNextSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrevious</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>activatePreviousSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCascade</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>cascadeSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTile</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>tileSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_as</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mdiArea</sender>
   <signal>subWindowActivated(QMdiSubWindow*)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateChipset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mdiArea</sender>
   <signal>subWindowActivated(QMdiSubWindow*)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateMenus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuWindow</sender>
   <signal>aboutToShow()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateWindowMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mdiArea</sender>
   <signal>subWindowActivated(QMdiSubWindow*)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateActionMdi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChange_dimension</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeDimension()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clearMap()</slot>
  <slot>undo()</slot>
  <slot>redo()</slot>
  <slot>openMap()</slot>
  <slot>maximizeMap()</slot>
  <slot>minimizeMap()</slot>
  <slot>lowerLayer()</slot>
  <slot>higherLayer()</slot>
  <slot>collisionLayer()</slot>
  <slot>visuaLayer()</slot>
  <slot>changeChipset()</slot>
  <slot>brushTool()</slot>
  <slot>paintPotTool()</slot>
  <slot>pipetteTool()</slot>
  <slot>save()</slot>
  <slot>gridLayer(bool)</slot>
  <slot>changeName()</slot>
  <slot>newMap()</slot>
  <slot>saveAs()</slot>
  <slot>updateMenus()</slot>
  <slot>updateChipset()</slot>
  <slot>updateWindowMenu()</slot>
  <slot>updateActionMdi()</slot>
  <slot>changeDimension()</slot>
 </slots>
</ui>
